scripts/mkcloud

```diff
@@ -482,6 +491,13 @@ function setuphost()
         [[ $arch = s390x ]] && {
             echo 1 > /proc/sys/vm/allocate_pgste
         }
+
+        # enable UEFI boot
+        [[ -n $want_efi ]] && {
```

Comment:
either we rename that to want_x86_64_efi or we need to add another $arch condition here..